src/main/java/teammates/storage/api/QuestionsDb.java

```diff
@@ -0,0 +1,765 @@
+    }
+    
+    /**
+     * Updates the feedback question identified by `{@code newAttributes.getId()}
+     *   and changes the {@code updatedAt} timestamp to be the time of update.
+     * For the remaining parameters, the existing value is preserved
+     *   if the parameter is null (due to 'keep existing' policy).<br>
+     * 
+     * Preconditions: <br>
+     * * {@code newAttributes.getId()} is non-null and
+     *  correspond to an existing feedback question. <br>
+     */
+    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes)
+            throws InvalidParametersException, EntityDoesNotExistException {
+        updateFeedbackQuestion(newAttributes, false);
+    }
+    
+    /**
+     * Updates the feedback question identified by `{@code newAttributes.getId()}
+     * For the remaining parameters, the existing value is preserved
+     *   if the parameter is null (due to 'keep existing' policy).<br>
+     * The timestamp for {@code updatedAt} is independent of the {@code newAttributes}
+     *   and depends on the value of {@code keepUpdateTimestamp}
+     * Preconditions: <br>
+     * * {@code newAttributes.getId()} is non-null and
+     *  correspond to an existing feedback question. <br>
+     */
+    public void updateFeedbackQuestion(FeedbackQuestionAttributes question, boolean keepUpdateTimestamp)
+            throws InvalidParametersException, EntityDoesNotExistException {
+        
+        Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, question);
+        if (!question.isValid()) {
+            throw new InvalidParametersException(question.getInvalidityInfo());
+        }
+        
+        updateQuestionWithoutFlushing(question, keepUpdateTimestamp);
+        
+        log.info(question.getBackupIdentifier());
+        getPm().close();
+    }
+    
+    /**
+     * Updates the given feedback question. Does not flush or commit therefore any code calling this
+     * must handle flushing or committing.
+     * @param question
+     * @throws InvalidParametersException
+     * @throws EntityDoesNotExistException
+     */
+    public FeedbackQuestionAttributes updateFeedbackQuestionWithoutComitting(
+            FeedbackQuestionAttributes question)
+            throws InvalidParametersException, EntityDoesNotExistException {
+        return new FeedbackQuestionAttributes(
+                     updateQuestionWithoutFlushing(question, false));
+    }
+
+    /**
+     * Updates the given feedback question. Does not flush or commit therefore any code calling this
+     * must handle flushing or committing.
+     * @param question
+     * @param keepUpdateTimestamp
+     * @throws InvalidParametersException
+     * @throws EntityDoesNotExistException
+     */
+    public Question updateQuestionWithoutFlushing(FeedbackQuestionAttributes question,
+            boolean keepUpdateTimestamp) throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
Could write the 2 parameters on a single line and the 2 exceptions on another line